lib/practice_exercises.rb

```diff
@@ -1,13 +1,46 @@
-
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n^2)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  list.each_with_index do |value, index|
+    if index > 0 
+      if value == list[index - 1] 
+        list.slice!(index) 
+      end
+    end
+  end
+  return list
 end
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
```

Comment:
Since you have those two nested loops you actually have a time complexity of O(n<sup>2</sup>)